src/main/java/net/atos/zac/app/zaken/ZakenRESTService.java

```diff
@@ -236,6 +240,20 @@ public RESTZaakOverzicht toekennenAanIngelogdeMedewerkerVanuitLijst(final RESTZa
         return zaakOverzichtConverter.convert(zaak);
     }
 
+    @PUT
+    @Path("toekennen/groep")
+    public RESTZaak groepToekennen(final RESTZaakToekennenGegevens restZaakToekennenGegevens) {
+        final Zaak zaak = zrcClientService.readZaak(restZaakToekennenGegevens.uuid);
+        final List<Rol<?>> rollen = zrcClientService.listRollen(zaak.getUrl());
+
+        final Group group = flowableService.readGroup(restZaakToekennenGegevens.groepId);
+        rollen.add(bepaalRolGroep(group, zaak));
+
+        zrcClientService.updateRollen(zaak.getUrl(), rollen);
```

Comment:
Maak van de 2 calls zrcClientService.listRollen(), zrcClientService.updateRollen(zaak.getUrl(), rollen); 1 nieuwe methode zrcClientService.updateRol(zaak.getURL(), bepaalRolGroep(group, zaak))

Bedenk daarbij dat de zaak al aan een groep kan zijn toegekend. Zo ja moet deze dan niet eerst verwijderd worden? Ik verwacht dat de logica dus iets complexer is.

Reply:
De logica omtrent het verwijderen/toevoegen zit in de updateRollen methode

Reply:
Ik heb Update en delete rol methodes toegevoegd aan de zrcClientService en een stuk gerefactored voor groep en medewerker